cloudinit/net/activators.py

```diff
@@ -26,20 +39,31 @@ def available() -> bool:
     def bring_up_interface(device_name: str) -> bool:
         raise NotImplementedError()
 
+    @staticmethod
+    @abstractmethod
+    def bring_down_interface(device_name: str) -> bool:
+        raise NotImplementedError()
```

Comment:
It'd be nice in these methods if we docstr that True means success and False means failure to perform the operation.